Style.css

```
@@ -1,43 +1,40 @@
 @import url('https://fonts.googleapis.com/css2?family=PT+Serif&display=swap');
+
 body {
-  margin: 0;
+  margin: 15px;
```

Use 16px as a base font size as this is the one most commonly used. It surely may vary.

```suggestion
  margin: 16px;
```